src/ContaoManager/Plugin.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Terminal42\ContaoAutoRegistrationBundle\ContaoManager;
```

Comment:
Please change the namespace to `Terminal42\AutoRegistrationBundle` (no `Contao`).